app/views/expense/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<%= stylesheet_link_tag "expenses" %>
+<div class='navbar navbar-login'>
+  <%= link_to 'Back', category_index_path, class: 'back-link' %>
+  <h2 style="color: white; background-color: transparent;">EXPENSES</h2>
+  <%= button_to "Sign out", destroy_user_session_path, method: :delete, class: "sign_out text_white" %>
+</div>
+ <div class="total">
+    <h4>TOTAL</h4> <strong> $<%=@category.total_amount%></strong> 
+ </div>
+
+<section class="product-section">
+  <div class='icon-container'>
+    <img src="<%= @category.icon %>" class="category-icon">
+    <h2 class="text_white"><%= @category.name %></h2>
+  </div>
+
+  <div class="exchanges">
+    <% if @expenses.length == 0 %>
+      <p class="empty_data text-center">No expense yet. Add a new one!!!</p>
+    <% else %>
+      <% @expenses.each do |expense| %>
+        <%= render expense %>
+      <% end %>
+    <% end %>
+  </div>
+
+ 
+  <div class="new_category_btn">
+    <%= link_to "New expense", new_category_expense_path(@category), class: "fcc-btn" %>
```

Comment:
- [ ] As mentioned by the previous reviewer kindly follow the given design 🙃 

Your design

<img width="374" alt="Screenshot 2023-09-16 at 19 27 49" src="https://github.com/yosaddis/ruby-on-rails-capstone/assets/74501523/455e9251-dde5-454a-b19e-7cfcf13b3265">